schema/parser.go

```diff
@@ -413,6 +413,19 @@ func parseIdentity(opt *sqlparser.IdentityOpt) string {
 	return strings.ToUpper(opt.Behavior)
 }
 
+func parseDefaultDefinition(opt *sqlparser.DefaultDefinition) *DefaultDefinition {
+	if opt == nil || opt.Value == nil {
+		return nil
+	}
+	defaultVal := parseValue(opt.Value)
+
+	name := "DEFAULT"
```

Comment:
Also, you called it `DefaultDefinition#name`, but is `DEFAULT` really a "name" of the default? I think this naming is partly related to my confusion https://github.com/k0kubun/sqldef/pull/123/files#r650809354.

Reply:
This naming may have been inappropriate.
An appropriate name might be empty, or `DF_constraint`, etc.